test/flows/check_uk_visa_flow_test.rb

```diff
@@ -9,6 +9,7 @@ class CheckUkVisaFlowTest < ActiveSupport::TestCase
   setup do
     testing_flow CheckUkVisaFlow
     @electronic_visa_waiver_country = "kuwait"
+    @electronic_travel_authorisation_country = "qatar"
```

Comment:
Are the countries in some enum somewhere? If yes, can we reuse it? If not, can we create a tech task to do it so that we can make changes in a single place if/when a country name changes?

Reply:
Yes, but it does not make sense to use them in this test as the tests only require a single country, not a list of countries